### PR TITLE
Contributor mailer

### PR DESCRIPTION
**Pull request by saracarl**

### CODE REVIEW

**Comment by saracarl** [comment]
Here's the question -- when I want to set this up for a particular customer/collection owner, how am I going to do that?  This mailer uses hardcoded ADMIN_EMAILS for the "to".  I'd like to either set it to the collection owner's email address OR pass it in via the rake task to this mailer.

File: app/mailers/admin_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  before_filter :add_inline_attachments!
+
+  default from: "FromThePage <support@fromthepage.com>"
+  layout "admin_mailer"
+  
+  def contributor_stats(collection_id, start_date, end_date)
+
+    new_contributors(collection_id, start_date, end_date)
+
+    mail from: SENDING_EMAIL_ADDRESS, to: ADMIN_EMAILS, subject: "New Transcription Information "
```
Comment (saracarl) [inline app/mailers/admin_mailer.rb]: hard coded ADMIN_EMAILS here.  Can we pull the owner from the collection_id?

File: lib/tasks/contributor_stats.rake [hunk 2]
```diff
@@ -0,0 +1,10 @@
+namespace :fromthepage do
+  desc "transcription stats from the previous 24 hours"
+  task :contributor_stats, [:collection_id] => :environment do |t,args|
```
Comment (saracarl) [inline lib/tasks/contributor_stats.rake]: How hard would it be for the rake task to take one (or more) email addresses and use them for the "to" in the mailer?  That's most flexible.

File: app/assets/stylesheets/datepicker/demo.css [hunk 3]
```diff
@@ -0,0 +1,234 @@
+body
```
Comment (saracarl) [inline app/assets/stylesheets/datepicker/demo.css]: Is this css file actually being used anywhere?  If not, let's not include it.

File: lib/tasks/contributor_stats.rake (hunk 2 above)
Comment (tbl73) [inline lib/tasks/contributor_stats.rake]: It's very easy to set it to take one email; I think the "or more" isn't bad, but I'm investigating.

File: app/assets/stylesheets/datepicker/demo.css (hunk 3 above)
Comment (tbl73) [inline app/assets/stylesheets/datepicker/demo.css]: It doesn't look like it is, so I just removed it from my environment.

File: app/mailers/admin_mailer.rb (hunk 1 above)
Comment (tbl73) [inline app/mailers/admin_mailer.rb]: You probably can - do you want to send it to the collection owner or pass it in the rake task?

**Pull request merged**